src/freenet/node/NewPacketFormat.java

```diff
@@ -992,7 +992,7 @@ public void checkForLostPackets() {
 	 * queued at plus the lesser of half the RTT or 100ms if there are acks queued. 
 	 * Otherwise Long.MAX_VALUE to indicate that we need to get messages from the queue. */
 	@Override
-	public long timeNextUrgent(boolean canSend) {
+	public long timeNextUrgent(boolean canSend, long now) {
 		long ret = Long.MAX_VALUE;
```

Comment:
In short: We don’t know whether anyone uses this. But transport plugins just might — which are already bitrotting, so this could make the bitrot worse.

Is it necessary to do this by changing the function?